<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewModelWizard</class>
 <widget class="QDialog" name="NewModelWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asistente de Creación de Nuevo Modelo</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblStep1Title">
         <property name="text">
          <string>&lt;h2&gt;Paso 1: Ubicación del modelo&lt;/h2&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblStep1Info">
         <property name="text">
          <string>Seleccione dónde desea guardar el nuevo archivo de modelo (.pdb).</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_1">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <widget class="QLineEdit" name="edPdbFile">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Ruta del archivo .pdb...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowsePdb">
           <property name="text">
            <string>Examinar...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>300</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lblStep2Title">
         <property name="text">
          <string>&lt;h2&gt;Paso 2: Información básica del modelo&lt;/h2&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblStep2Info">
         <property name="text">
          <string>Ingrese la información básica del modelo.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblNombre">
           <property name="text">
            <string>&lt;b&gt;Nombre del modelo:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edNombre">
           <property name="placeholderText">
            <string>Ej: Acelerador Lineal</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblFecha">
           <property name="text">
            <string>Fecha de inicio:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="edFecha">
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblTecnica">
           <property name="text">
            <string>Técnica:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edTecnica">
           <property name="placeholderText">
            <string>Ej: Radioterapia</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblAnalistas">
           <property name="text">
            <string>Analistas:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="edAnalistas">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Nombres de los analistas</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblInstitucion">
           <property name="text">
            <string>Institución/Hospital:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edInstitucion">
           <property name="placeholderText">
            <string>Nombre de la institución</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblDescripcion">
           <property name="text">
            <string>Descripción:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPlainTextEdit" name="edDescripcion">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Descripción del modelo</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblObservaciones">
           <property name="text">
            <string>Observaciones:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPlainTextEdit" name="edObservaciones">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Observaciones adicionales</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lblStep3Title">
         <property name="text">
          <string>&lt;h2&gt;Paso 3: Etapas del proceso&lt;/h2&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblStep3Info">
         <property name="text">
          <string>¿Quiere añadir las etapas del proceso?</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAddStage">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Añadir Etapa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveStage">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Eliminar Etapa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveUp">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Subir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveDown">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Bajar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeStages">
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Código (solo etapas)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="lblStep4Title">
         <property name="text">
          <string>&lt;h2&gt;Paso 4: Creación del modelo&lt;/h2&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCreationStatus">
         <property name="text">
          <string>Preparando la creación del modelo...</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>300</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
